.github/workflows/scripts/slack_notifications/gem_notifier.rb

```diff
@@ -0,0 +1,83 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'time'
+require 'httparty'
+require_relative 'slack_notifier'
+
+class GemNotifier < SlackNotifier
+  def self.check_for_updates(watched_gems)
+    return if gem_list_empty(watched_gems)
+    watched_gems.each do |gem_name|
+      verify_gem(gem_name) ? gem_info = verify_gem(gem_name) : return
+      versions = gem_versions(gem_info)
+      notifier.send_slack_message(gem_message(gem_name, versions)) if gem_updated?(versions)
+    end
+  end
+
```

Comment:
Same comment here as with the CVE notifier. All methods below `check_for_updates` could be marked as private.

Reply:
Now marked as private methods: [here for gem_notifier](https://github.com/newrelic/newrelic-ruby-agent/pull/1432/files#diff-ddcde05fb3aba0b9770eddeb6be0d8e2a76ac7f86d69786141db8cb712412794R24) and [here for cve_notifier](https://github.com/newrelic/newrelic-ruby-agent/pull/1432/files#diff-8ca3366eee754ac705491b7f94a5df5a8a546e1280ede02134a37e2c0392158fR25)